packages/web-extension/src/keyAgent/consumeKeyAgent.ts

```diff
@@ -1,3 +1,6 @@
+import '@cardano-sdk/util';
+import '@cardano-sdk/wallet/dist/cjs/KeyManagement';
+import 'rxjs';
```

Comment:
We need to do something about these imports, because they will be impacting end-users. One of the runtime environments of SDK is browser via bundlers. Bundle size matters and having such imports will prevent bundlers from doing tree-shaking (just include the entire libraries instead of selectively picking only the exports that are used)